elephant/sta.py

```diff
@@ -314,3 +320,59 @@ def spike_field_coherence(signal, spiketrain, **kwargs):
 
     return (pq.Quantity(sfc, units=pq.dimensionless),
             pq.Quantity(frequencies, units=pq.Hz))
+
+
+def spike_triggered_population_response(spiketrains,
+                                        bin_size=1*pq.ms,
+                                        sigma=5*pq.ms):
+    """
+    Spike triggered population response (stPR) as described in [1]_.
+
+    Parameters
+    ----------
+    spiketrains : list of neo.SpikeTrain
+        list of spike train objects. The spike trains should have the same
+        `t_start` and `t_stop` times.
+
```

Comment:
Don't forget to mention it in my last review:
There should not be any blank line between parameters in the description.